src/components/Tooltip.tsx

```diff
@@ -0,0 +1,13 @@
+import { Tooltip as MuiTooltip } from "@material-ui/core";
+import React from "react";
+
+export class Tooltip extends React.Component<any, any> {
```

Comment:
make it as functional component 
also avoid using `any`

Reply:
Ok, I tried to make it a functional component, seems to work.